### PR TITLE
Task 9 djangorest

### PR DESCRIPTION
**Pull request by VasiaBirchak**

### CODE REVIEW

File: myblog/blog/api/serializers.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from blog.models import BlogPost
+
+
+class PostSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = BlogPost
+        fields = ['title', 'body', 'author']
```
Comment (ejariza-evowill) [inline myblog/blog/api/serializers.py:8]: As this is not going to change during runtime change it to tuples.

File: myblog/blog/test_rest.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db
+class TestPostEndpoint(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', password='1234')
+        BlogPost.objects.create(title='Post 1', body='Body of post 1', author=self.user)
+        BlogPost.objects.create(title='Post 2', body='Body of post 2', author=self.user)
+        BlogPost.objects.create(title='Post 3', body='Body of post 3', author=self.user)
```
Comment (ejariza-evowill) [inline myblog/blog/test_rest.py:14]: This can be a comprehension list

**Review by ejariza-evowill — CHANGES_REQUESTED** [review]

File: myblog/blog/test_rest.py (hunk 2 above)
Comment (VasiaBirchak) [reply myblog/blog/test_rest.py:14]: Good idea

File: myblog/blog/api/serializers.py (hunk 1 above)
Comment (VasiaBirchak) [reply myblog/blog/api/serializers.py:8]: Thank you, this is a good solution

**Comment by ejariza-evowill** [comment]
Remember to run flake8 command locally so you can see style errors before you push

**Comment by VasiaBirchak** [comment]
Sorry, I forgot about the flake and remembered it only after I pushed it, I just wanted to change it and you already added a comment)

**Review by ejariza-evowill — APPROVED** [review]

**Pull request merged**